network/p2p/consumers.go

```diff
@@ -33,14 +33,17 @@ type InvCtrlMsgNotif struct {
 	Error error
 	// MsgType the control message type.
 	MsgType p2pmsg.ControlMessageType
+	// Count the number of errors.
+	Count uint64
 }
 
 // NewInvalidControlMessageNotification returns a new *InvCtrlMsgNotif
-func NewInvalidControlMessageNotification(peerID peer.ID, ctlMsgType p2pmsg.ControlMessageType, err error) *InvCtrlMsgNotif {
+func NewInvalidControlMessageNotification(peerID peer.ID, ctlMsgType p2pmsg.ControlMessageType, err error, count uint64) *InvCtrlMsgNotif {
```

Comment:
To prevent misuse, please document the parameters and their purpose.

Reply:
https://github.com/onflow/flow-go/pull/4773/commits/a3edb0855652e4000374841f1dbbf5b9c753c6e3